docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -255,3 +255,21 @@ image::images/agent-output-settings.png[Screen capture showing the {ls} output p
 
 Any {agent}s enrolled in the agent policy will begin sending data to the
 specified outputs.
+
+[discrete]
+[[add-fleet-server-to-policy]]
+== Add a {fleet-server} to a policy
```

Comment:
@elastic/obs-docs I could use another set of 👀 here...

The [docs build is failing](https://elasticsearch-ci.elastic.co/job/elastic+observability-docs+pull-request+build-docs/2877/console) on this PR with the error `invalid reference: add-fleet-server-to-policy`, but `add-fleet-server-to-policy` _does_ exist (above). I can't figure out what I'm missing...  🤔

Reply:
@colleenmcginnis It looks like the link is in a shared region. Have you tried using the external link format to see if that fixes the problem?